dataUtil/constants.py

```diff
@@ -0,0 +1,2 @@
+MS = 1000
+SEG_LEN = 2.5
```

Comment:
might also consider using more descriptive names e.g. 
SEGMENT_LENGTH instead of SEG_LEN so we don't forget what they mean